pontoon/base/views.py

```diff
@@ -988,50 +993,75 @@ def transvision(request):
         return HttpResponse("error")
 
 
+def get_download_content(slug, code, relative_path):
+    """
+    Get content for the file to be downloaded.
+
+    :param str slug: Project slug.
+    :param str code: Locale code.
+    :param str relative_path: Resource path.
+    """
+    # Clone or update project repository
+    project = get_object_or_404(Project, slug=slug)
+    project.repositories.first().pull()
```

Comment:
This doesn't work if the source files and the files from the locale we want aren't in the first repo. Technically we have to pull from all the repos for the project in order to be sure we have both the source strings and the locale strings.